### PR TITLE
Revamped design

### LINKED ISSUE #21: Update contributing guidelines
With the new website structure, the contributing guidelines are now outdated. 

A fix for this issue should address all the new requirements for contributing.

---

> ✅️ By submitting this issue, I have verified the following

* [x] Checked to see if the issue has already been discussed before 🤔️
* [x] If proposing new content to be added, made sure enough details were provided 🔍️

### PR DESCRIPTION
**Pull request by fvcproductions**
Based off #29 

Fixes #21 #22 

- adds issue templates for adding, removing, editing apprenticeships
- favicons and branding images
- SEO
- green theme instead to show "growth"

---

## ✅️ By submitting this PR, I have verified the following

* [x] Checked to see if a similar PR has already been opened 🤔️
* [x] Reviewed the contributing guidelines 🔍️
* [x] Added my name to the bottom of the list under the **Credits** section in the `README.md` with a link to my website or GitHub profile 👥️

### CODE REVIEW

**Comment by fvcproductions** [comment]
This pull request **fixes 1 alert** when merging dc5fd548d621370d0ed35131291574b29d0a0829 into 66dfe577c03083a4d3ba23f6aa6bb74ab03f878e - [view on LGTM.com](https://lgtm.com/projects/g/fvcproductions/apprenticeships.me/rev/pr-8ef6dc32725f5c82774cc08b3caddfcb0f368683)

**fixed alerts:**

* 1 for Call to eval\-like DOM function

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by fvcproductions** [comment]
This pull request **fixes 1 alert** when merging 2166be88dbd794d0f369e4008c1f32155c06f4ac into 66dfe577c03083a4d3ba23f6aa6bb74ab03f878e - [view on LGTM.com](https://lgtm.com/projects/g/fvcproductions/apprenticeships.me/rev/pr-cb86d3559ce0c27aaab6042f4382be801d61a250)

**fixed alerts:**

* 1 for Call to eval\-like DOM function

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by fvcproductions** [comment]
This pull request **fixes 1 alert** when merging 9a427652ba7c1feaff1e9d5c0f5ab049b53f4f00 into 66dfe577c03083a4d3ba23f6aa6bb74ab03f878e - [view on LGTM.com](https://lgtm.com/projects/g/fvcproductions/apprenticeships.me/rev/pr-d44c8cfe852639ab14bb1e0bb1aed68e804a50b5)

**fixed alerts:**

* 1 for Call to eval\-like DOM function

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by fvcproductions** [comment]
This pull request **fixes 1 alert** when merging bd230f5be04ad6d76f3dc48285c96491688181ac into 66dfe577c03083a4d3ba23f6aa6bb74ab03f878e - [view on LGTM.com](https://lgtm.com/projects/g/fvcproductions/apprenticeships.me/rev/pr-d732cff83c3d689b15df89a338301cf73d7ea6df)

**fixed alerts:**

* 1 for Call to eval\-like DOM function

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by fvcproductions** [comment]
This pull request **fixes 1 alert** when merging 2b6c88b81782b60eec2ddbb94e362dfb4a24bfaa into 66dfe577c03083a4d3ba23f6aa6bb74ab03f878e - [view on LGTM.com](https://lgtm.com/projects/g/fvcproductions/apprenticeships.me/rev/pr-fd19554eb233f959bd78a947925b8abce164d0df)

**fixed alerts:**

* 1 for Call to eval\-like DOM function

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by fvcproductions** [comment]
@benjaminmodayil This is ready to be reviewed - let me know what you think!

**Comment by fvcproductions** [comment]
In the future, we might want to also add a featured image for each apprenticeship which would then make the layout look something like this:

localhost_1234_

**Comment by fvcproductions** [comment]
This pull request **fixes 1 alert** when merging 7860c64dec73b9e7416fdf5b24a2bf80df24fc86 into 66dfe577c03083a4d3ba23f6aa6bb74ab03f878e - [view on LGTM.com](https://lgtm.com/projects/g/fvcproductions/apprenticeships.me/rev/pr-c38b54b2fb2b60dd7a9a4ebfa43fa6b758eba050)

**fixed alerts:**

* 1 for Call to eval\-like DOM function

---

*Comment posted by [LGTM.com](https://lgtm.com)*

File: static/index.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
 // var instantsearch = require("instantsearch.js");
-import "./components/search";
-import "./components/github";
-import "./components/scroll-nav";
+require('./components/search');
+require('./components/github');
```
[the remark this answers is not in the document]
Comment (benjaminmodayil) [reply static/index.js]: Is it preferred to use requires versus imports? I've always been confused about this.

**Comment by benjaminmodayil** [comment]
@fvcproductions 
Definitely a lot cleaner and I'm glad you decided to remove that floating navigation. It was pretty troublesome.

Things I've noticed:

1. The shadow's blur around the logo makes the logo appear a little blurry, might want to play with that shadow a little bit more.
2. The listings for the search should probably be centered when in a single-column (mobile) versus left aligned.
3. The items in the image need a different breakpoint (around 770px wide)awkward breakpoint]
4. The navigation also stacks a little oddly on mobile: navigation stacking

I'll leave these up to your discretion before merging.

**Comment by fvcproductions** [comment]
@benjaminmodayil 

1. Are we talking about the plant logo?

screenshot 2019-01-13 15 27 24

2. I think it looks better left aligned on mobile since the other items up top are left aligned as well.

screenshot 2019-01-13 15 28 55
screenshot 2019-01-13 15 29 12

3. Changed this to be 1 column on a tablet

screenshot 2019-01-13 15 30 41

4. Will fix this!

**Comment by benjaminmodayil** [comment]
1. Yes the plant logo. The image you sent back looks less blurry... Wonder if it might be a rendering thing on my end.

2.,3.,4. 👍

By the way, I realized that we're not running a command on build to update the listings for the algolia search. I just tried to run the command locally and it seems to return an error that I'll have to look into. I'll open an issue and assign it to myself and check it out tomorrow morning.

**Comment by fvcproductions** [comment]
This pull request **fixes 1 alert** when merging edc3739fbb8245575ce7a15b23b149da6a36d0d8 into 66dfe577c03083a4d3ba23f6aa6bb74ab03f878e - [view on LGTM.com](https://lgtm.com/projects/g/fvcproductions/apprenticeships.me/rev/pr-0cf8440e0bc6144e7d4cbc1e366fff010f703055)

**fixed alerts:**

* 1 for Call to eval\-like DOM function

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by fvcproductions** [comment]
@benjaminmodayil I am removing the comments from Codacy but beyond the Algolia build issue, does everything else look okay?

**Pull request merged**